server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/crud/InsertRowsStatement.java

```diff
@@ -136,4 +136,16 @@ public long getMinTime() {
   public Object getFirstValueOfIndex(int index) {
     throw new NotImplementedException();
   }
+
+  @Override
+  public InsertBaseStatement split() {
+    List<InsertRowStatement> mergedList = new ArrayList<>();
+    for (InsertRowStatement child : this.insertRowStatementList) {
+      List<InsertRowStatement> childSplitResult = child.getSplitList();
+      mergedList.addAll(childSplitResult);
+    }
+    InsertRowsStatement splitResult = new InsertRowsStatement();
+    splitResult.setInsertRowStatementList(mergedList);
+    return splitResult;
```

Comment:
Maybe add a should split check is better.

Reply:
added